ls1intum/Artemis
# `Programming exercises`: Update gradle template to 8.0.1 and newest SCA versions

### Checklist
#### General

### Motivation and Context
We should keep the template up to date.

### Description
Updates the template gradle version to 8.0.1 and the default plugin version to the newest releases.

Related docker PR: https://github.com/ls1intum/artemis-maven-docker/pull/125

### Steps for Testing
Create a new Programming Exercise (gradle and maven) with SCA and check that everything still works.

Verify that the template uses the same version as in the docker image.

### Review Progress

#### Code Review
#### Manual Tests
